extensions/wikia/PotentialMemberPageExperiments/templates/PMPEntryPoint.mustache

```diff
@@ -1,4 +1,4 @@
-<div class="pmp-entry-point pmp-entry-point-{{bannerType}} clearfix">
+<div class="pmp-entry-point pmp-entry-point-{{bannerType}}">
```

Comment:
Yup, that was the point - to use one template and just style it differently with this class. :)